docs/pages/installation/_index.md

```diff
@@ -5,20 +5,252 @@ weight: 2
-| [Kubernetes with Kubeadm](./platforms/k8s)                    | Yes                            |                                        |
-| [AWS Elastic Kubernetes Service (EKS)](./platforms/k8s)       | In Progress                    |                                        |
+### Quick Install
 
-#### Peer your clusters
+#### Liqoctl 
 
-When you have installed Liqo on your clusters, you can decide to peer them to offload your applications on a different cluster as documented in [usage section](/usage).
\ No newline at end of file
+Liqoctl is the swiss-knife CLI tool to install and manager Liqo. 
+It is strongly recommended installing Liqo using Liqoctl since it provides customizations for supported providers.
+
+To install Liqoctl, first, you have to set the architecture and OS of your host:
+
+```bash
+#(Draft)
+OS=linux # possible values: linux,windows,darwin
+ARCH=amd64 # possible values: amd64,arm64 
+```
+
+Then, you should execute the following commands the latest version of liqoctl:
+```
+LATEST_RELEASE=$(curl -L -s -H 'Accept: application/json' https://github.com/liqotech/liqo/releases/latest)
+LATEST_VERSION=$(echo $LATEST_RELEASE | sed -e 's/.*"tag_name":"\([^"]*\)".*/\1/')
+curl --fail -LSO "https://github.com/liqotech/liqo/releases/download/${LATEST_VERSION}/liqoctl-${OS}-${ARCH}" && chmod +x liqoctl && sudo cp liqoctl-${OS}-${ARCH} /usr/bin/liqoctl
+```
+
+Obviously, you can obtain the same result by visiting the [Liqo releases](https://github.com/liqotech/liqo/releases/) page on our project.
+
+#### Pre-Requirements
+
+{{% notice note %}}
+Liqo only supports Kubernetes >= 1.19.0.
+{{% /notice %}}
+
+According to your cluster provider, you may have to perform simple steps before triggering the installation process:
+
+{{< tabs >}}
+{{% tab name="K8s (Kubeadm)" %}}
+
+##### Supported CNIs
+
+Liqo supports K8s clusters using the following CNIs:
+* [Weave](https://github.com/weaveworks/weave)
+* [Flannel](https://github.com/coreos/flannel)
+* [Canal](https://docs.projectcalico.org/getting-started/kubernetes/flannel/flannel)
+* [Calico](https://www.projectcalico.org/)
+
+##### Configuration 
+
+You only have to export the KUBECONFIG environment variable. 
+Otherwise, liqoctl will try to use the kubeconfig in kubectl default path (i.e. `${HOME}/.kube/config` )
+
+```bash
+export KUBECONFIG=/your/kubeconfig/path
+```
+
+{{% /tab %}}
+
+{{% tab name="EKS" %}}
+
+##### Supported CNIs
+
+Liqo supports EKS clusters using the default CNI:
+* [AWS EKS - amazon-vpc-cni-k8s](https://github.com/aws/amazon-vpc-cni-k8s)
+
+##### Configuration
+
+Compared to the other K8s managed providers, Liqo leverages AWS credentials to authenticate peered clusters.
+Therefore, Liqo on AWS requires some extra permissions.
+In addition to the read-only permissions used to configure the cluster installation, Liqo uses AWS users to map peering access
+to EC2 cluster.
```

Comment:
```suggestion
Liqo leverages AWS credentials to authenticate peered clusters.
Specifically, in addition to the read-only permissions used to configure the cluster installation (i.e., retrieve the appropriate parameters), Liqo uses AWS users to map peering access to EKS clusters.
```

Would avoid to compare with other providers, as it might introduce confusion if the big picture is not clear.